### DISCUSSION TITLE
Widget color

### DISCUSSION THREAD

**Discussion by mtldf** [iSoron/uhabits#1248] [Feature Requests]
Right now the widgets have a dark theme, for example the checkmark one is black, and when the task is completed it turns to whatever color is set for that specific habit (see screenshot). This happens no matter the theme used in the application or in the system. Would it be possible to have a white version too? (when unchecked) 
![Screenshot_20211229-225434_Nova7](https://user-images.githubusercontent.com/96846614/147706424-78440b1a-a2e4-4b0e-a643-924d291b9d65.jpg)

**Comment by brimwats1** [comment]
big +1 to this idea, it's nearly impossible to see the widget edges on pop-up widget

![Screenshot_20211229-180956_Popup Widget](https://user-images.githubusercontent.com/49103046/147716114-ef5bbc0d-4a72-406e-9444-4bacd971a98e.jpg)
